docker/compose/dev-split/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  redis:
+    container_name: "redis"
+    image: "redis:3.0.6-alpine"
+    ports:
+      - "6379:6379"
+    expose:
+      - 6379
+  blazegraph:
+    container_name: "blazegraph"
+    build:
+      context: ../../..
+      dockerfile: docker/image/whyis-jetty/Dockerfile
+    image: tetherlessworld/whyis-blazegraph
```

Comment:
These are implicitly :latest tag. That's fine for now, we may want to rethink later.